src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+/**
+ * Formatter for Text UI of the application.
+ */
+public class Formatter {
+	
+	/** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+    
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+    /** Formats a list of strings as a viewable indexed list. */
```

Comment:
i don't think you have extracted all methods here? try to find out what more can be done